sql-infra: add resource control feature (#12353)
pingcap/docs: sql-statements/sql-statement-show-create-resource-group.md

---
title: SHOW CREATE RESOURCE GROUP
summary: Learn the usage of SHOW CREATE RESOURCE GROUP in TiDB.
---

# SHOW CREATE RESOURCE GROUP

You can use the `SHOW CREATE RESOURCE GROUP` statement to view the current definition of a resource group.

## Synopsis

```ebnf+diagram
ShowCreateResourceGroupStmt ::=
    "SHOW" "CREATE" "RESOURCE" "GROUP" ResourceGroupName

ResourceGroupName ::=
    Identifier
```

## Examples

> View the current definition of the resource group `rg1`.

Author: ```suggestion
Create a resource group `rg1` and then view the definition of `rg1`.
```

```sql
CREATE RESOURCE GROUP rg1 RU_PER_SEC=100;
Query OK, 0 rows affected (0.10 sec)
```

> 

Author: ```suggestion

View the definition of `rg1`.

```

```sql
SHOW CREATE RESOURCE GROUP rg1;
***************************[ 1. row ]***************************

> Resource_Group        | rg1
> Create Resource Group | CREATE RESOURCE GROUP `rg1` RU_PER_SEC=100
> 1 row in set (0.00 sec)

Author: ```suggestion
+----------------+--------------------------------------------+
| Resource_Group | Create Resource Group                      |
+----------------+--------------------------------------------+
| rg1            | CREATE RESOURCE GROUP `rg1` RU_PER_SEC=100 |
+----------------+--------------------------------------------+
1 row in set (0.01 sec)
```
